packages/expo/src/hooks/useSSO.ts

```diff
@@ -47,9 +49,11 @@ export function useSSO() {
      * to include the `rotating_token_nonce` on SSO callback
      * @ref https://clerk.com/docs/reference/backend-api/tag/Redirect-URLs#operation/CreateRedirectURL
      */
-    const redirectUrl = AuthSession.makeRedirectUri({
-      path: 'sso-callback',
-    });
+    const redirectUrl =
+      startSSOFlowParams.redirectUrl ??
+      AuthSession.makeRedirectUri({
+        path: 'sso-callback',
+      });
```

Comment:
❓ Should redirectUrl be any string ? Should we be more strict and do 
```suggestion
    const redirectUrl =
      AuthSession.makeRedirectUri({
        path: startSSOFlowParams.redirectUrl ?? 'sso-callback',
      });
```